cfme/rest/__init__.py

```diff
@@ -339,3 +340,30 @@ def _finished():
             collection.action.delete(*delete_entities)
 
     return entities
+
+
+def templates(request, rest_api, a_provider, vm, num=1):
```

Comment:
I think there a few things I'd change here
1) This function's name is misleading, it should probably be called, ```templatize_vm``` or something else, please add a docstring to it also
2) It should be split up between two functions, there should be a fixture in the ```test``` file. We shouldn't require a ```request``` object and definitely shouldn't ```addfinalizer``` inside this module.
3) In splitting it up, the iteration can happen in the fixture, not in this function.